dll/win32/advapi32/advapi32.spec

```diff
@@ -654,7 +654,7 @@
 @ stdcall SystemFunction040(ptr long long) # RtlEncryptMemory
 @ stdcall SystemFunction041(ptr long long) # RtlDecryptMemory
 @ stdcall -version=0x502 TraceEvent(double ptr) ntdll.EtwTraceEvent
-@ stdcall -stub -version=0x600+ TraceEvent(double ptr)
+@ stdcall -version=0x600+ TraceEvent(double ptr) EtwTraceEvent
 @ stdcall TraceEventInstance(double ptr ptr ptr) ntdll.EtwTraceEventInstance
```

Comment:
These (Etw)TraceEventInstance, and the TraceMessage ones from below, stay in ntdll even on Vista+ ? (and the redirection still needed then?)

Reply:
yes